.github/workflows/build-docker-image.yaml

```
@@ -0,0 +1,199 @@
+          GOOGLE_ACCESS_TOKEN: ${{ steps.gcp.outputs.access_token }}
+          IMAGE_TARGET: ${{ inputs.docker_gcp_registry }}/${{ inputs.docker_image_name }}:${{ steps.version.outputs.docker_debug_version_platform }}
+          SKOPEO_INSECURE_POLICY: "1"
+      - name: Build latest docker image - ${{ inputs.docker_image_name }}:latest-${{ steps.version.outputs.docker_platform }}
+        if: inputs.version_type == 'pr'
+        run: ${{ inputs.build_command }}
+        env:
+          GOOGLE_ACCESS_TOKEN: ${{ steps.gcp.outputs.access_token }}
+          IMAGE_TARGET: ${{ inputs.docker_gcp_registry }}/${{ inputs.docker_image_name }}:latest-${{ steps.version.outputs.docker_platform }}
+          SKOPEO_INSECURE_POLICY: "1"
+      - name: Log in to Docker Hub
+        if: inputs.version_type == 'release' && secrets.docker_hub_username != '' && secrets.docker_hub_token != ''
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)